Gopkg.lock

```diff
@@ -9,15 +9,9 @@
 
 [[projects]]
   branch = "master"
-  name = "github.com/Azure/go-ansiterm"
```

Comment:
Just to check - we understand and wanted this Azure delete at the same time?
I did a `dep ensure -update github.com/BurntSushi/toml` - and understand the numerous other changes that happen to the lock file are just part of what `dep` did (and I half consider that maybe a `dep` bug - we only asked to update a single package, but it went and did a bunch of other stuff as well, and afaik there is no way to tell it not to....).